Procfile

```diff
@@ -0,0 +1,3 @@
+# Procfile
+backend: bin/rails s -p 3000
+frontend: bin/webpack-dev-server
```

Comment:
Is this going to remain in place for the production deployment?  I only ask given the more NPM-bound approaches outlined using:
https://webpack.js.org/guides/production/#cli-alternatives

Reply:
Yes, this is a foreman config for developer convenience. Production uses the compiled code that doesn't require npm to start a separate webpack dev server.